src/lib/slider/slider.ts

```diff
@@ -213,21 +251,21 @@ export class MdSlider implements AfterContentInit, ControlValueAccessor {
     this.isSliding = true;
     this.isActive = true;
     this._renderer.addFocus();
-    this.updateValueFromPosition(event.center.x);
+    this._updateValueFromPosition(event.center.x);
   }
 
   /** TODO: internal */
   onSlideEnd() {
     this.isSliding = false;
-    this.snapThumbToValue();
   }
 
   /** TODO: internal */
   onResize() {
+    // Consider the slider to be sliding during resize to prevent animation.
     this.isSliding = true;
-    this._sliderDimensions = this._renderer.getSliderDimensions();
-    // Skip updating the value and position as there is no new placement.
-    this._renderer.updateThumbAndFillPosition(this._percent, this._sliderDimensions.width);
+    setTimeout(() => {
+      this.isSliding = false;
+    }, 0);
```

Comment:
I think this is not even necessary anymore now that the position is percent based.